ext/re2/extconf.rb

```diff
@@ -295,17 +295,18 @@ def build_with_system_libraries
   -labsl_crc32c
   -labsl_crc_internal
   -labsl_crc_cpu_detect
+  -labsl_raw_hash_set
   -labsl_hash
   -labsl_city
   -labsl_bad_variant_access
   -labsl_low_level_hash
-  -labsl_raw_hash_set
   -labsl_hashtablez_sampler
   -labsl_exponential_biased
   -labsl_bad_optional_access
   -labsl_str_format_internal
   -labsl_synchronization
   -labsl_graphcycles_internal
+  -labsl_kernel_timeout_internal
```

Comment:
I was hoping we could drop this workaround in https://github.com/mudge/re2/pull/96, but the runners are still using `pkgconf` 1.9.3. 2.0.0 will fix this: https://github.com/pkgconf/pkgconf/issues/268